scenario/features/privatedata.feature

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2021 IBM All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+Feature: Private data
+    Background:
+        Given I have deployed a Fabric network
+        And I have created and joined all channels
+        And I deploy golang chaincode named basic at version 1.0.0 for all organizations on channel mychannel with endorsement policy OR("Org1MSP.member","Org2MSP.member","Org3MSP.member")
+        And I create a gateway for user User1 in MSP Org1MSP
+        And I connect the gateway to peer0.org1.example.com
+        And I use the mychannel network
+        And I use the basic contract
+
+    Scenario: Submit to org1
+        When I prepare to submit an getPeerOrg transaction
```

Comment:
I hate to be that grammar lawyer but... "submit **a** getPeerOrg transaction"? 😉

Reply:
quite right :)